.travis.yml

```diff
@@ -88,6 +91,7 @@ matrix:
             - python-docutils
             - python-pygments
             - pngcrush
+            - libpoppler-cpp-dev
       language: node_js
       node_js:
         - node
```

Comment:
We don't build omega in this configuration - see a little below:

```
        # Bootstrap only xapian-core for emscripten build.
        - ./bootstrap xapian-core
```

Reply:
I didn't realize, I am going to remove it.